### PR TITLE
RHDEVDOCS-891: Reviewed the Workspace Data Model section for language…

### PR DESCRIPTION
**Pull request by supriya-bharadwaj**
… and asciidoc syntax.

Signed-off-by: Supriya Takkhi <sbharadw@redhat.com>

### What does this PR do?
Fixes lang. and mark-up issues in *Workspace Data Model -> https://www.eclipse.org/che/docs/workspace-data-model.html*.
File name: `src/main/pages/workspace-admin/workspace-data-model.adoc`
Hi @rkratky , can you please review this PR? Thanks!

### CODE REVIEW

File: src/main/pages/workspace-admin/workspace-data-model.adoc [hunk 1]
```diff
@@ -2,27 +2,42 @@
...
+|===
 
 [id="environment"]
-== Environment
+== Environment recipes
 
 image::workspaces/ws_data_model.png[]
 
-Environment recipe can also have content instead of location, if link:recipes.html[recipe] type is `dockerfile`, `compose` or `openshift`:
+Environment recipe can also have content instead of location, if the link:recipes.html[recipe] type is `dockerfile`, `compose` or `openshift`:
```
Comment (rkratky) [inline src/main/pages/workspace-admin/workspace-data-model.adoc]: serial comma
```suggestion
For link:recipes.html[recipe] types of `dockerfile`, `compose`, or `openshift`, content, not location, is specified.
```

File: src/main/pages/workspace-admin/workspace-data-model.adoc [hunk 2]
```diff
@@ -38,22 +53,22 @@ Environment recipe can also have content instead of location, if link:recipes.ht
 
 image::workspaces/ws_projects.png[]
 
-Project object structure is quite simple, `source.location` and `source.type` being the most important parameters. There are 2 importer types: `git` and `zip`. New location types can be provided by custom plugins, such as `svn`.
+The project object structure has the `source.location` and `source.type` parameters. There are two importer types: `git` and `zip`. New location types can be provided by custom plugins, such as `svn`.
 
-In the example above, a project does not have problems and mixins. If a project is misconfigured or missing sources, it will be marked with a problem (error code and a message explaining the problem).
+If a project is misconfigured or missing sources, it is marked with an error code and a message explaining the error. In the example above, the project does not have errors and mixins.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/workspace-data-model.adoc]: if
```suggestion
Incorrectly configured projects or projects missing sources are marked with error codes and messages explaining the error. In the example above, the project does not have errors and mixins.
```

File: src/main/pages/workspace-admin/workspace-data-model.adoc [hunk 3]
```diff
@@ -64,17 +79,17 @@ The `project` object can also include `source.parameters`, which is a map that c
...
-Here’s how commands can be saved in project configuration:
+The followling image shows ways to save commands in the project configuration.
 
 image::workspaces/project_commands.png[]
 
 [id="runtime"]
 == Runtime
 
-Runtime object is created when a workspace is in a running state. Runtime returns servers URLs (internal or external, depending on server config). Interested clients, like User Dashboard and IDE use these URLs.
+Runtime object is created when a workspace is in a running state. Runtime returns server URLs, internal or external, depending on the server configuration. Interested clients, like User Dashboard and IDE, use these URLs.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/workspace-data-model.adoc]:
```suggestion
A runtime object is created when a workspace is in a running state. Runtime returns server URLs, internal or external, depending on the server configuration. Interested clients, like the *User Dashboard* and the IDE, use these URLs.
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by supriya-bharadwaj** [comment]
Hi @rkratky , I have accepted your suggestions. Can we merge this branch, please? Thanks!

**Review by rkratky — APPROVED** [review]

**Pull request merged**